java/Cherepnin_Oleh/src/com/zoolatech/lecture4/tasks/_2/Node.java

```diff
@@ -0,0 +1,35 @@
+package com.zoolatech.lecture4.tasks._2;
+
+public class Node {
```

Comment:
the class should be private static and defined inside of the tree class - we don't need to expose an internal implementation of the tree to the rest of the world